src/pyfmi/fmi2.pyx

```diff
@@ -1185,7 +1185,6 @@ cdef class FMUModelBase2(FMI_BASE.ModelBase):
 
         self._t = start_time
         self._last_accepted_time = start_time
-        self._relative_tolerance = tolerance
```

Comment:
I removed this as it doesnt seem used anywhere else besides just getting a value assigned in setup_experiment.